tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py

```diff
@@ -309,29 +290,15 @@ def test_multi_objective_sample_independent_log_uniform_distributions() -> None:
     ) as mock2:
         mock1.return_value = attrs.value
         mock2.return_value = attrs.value
-        loguniform_suggestion = sampler.sample_independent(study, trial, "param-a", log_dist)
-    assert 1.0 <= loguniform_suggestion < 100.0
-    assert uniform_suggestion != loguniform_suggestion
-
-
-def test_multi_objective_sample_independent_disrete_uniform_distributions() -> None:
-    """Test samples from discrete have expected intervals."""
-
-    study = optuna.create_study(directions=["minimize", "maximize"])
-    random.seed(128)
+        float_suggestion = sampler.sample_independent(study, trial, "param-a", float_dist)
+    assert 1.0 <= float_suggestion < 100.0
 
-    disc_dist = optuna.distributions.FloatDistribution(1.0, 100.0, step=0.1)
-
-    def value_fn(idx: int) -> float:
-        return int(random.random() * 1000) * 0.1
-
-    past_trials = [
-        frozen_trial_factory(
-            i, [random.random(), random.random()], dist=disc_dist, value_fn=value_fn
-        )
-        for i in range(16)
-    ]
+    if float_dist.step == 0.1:
+        assert abs(int(float_suggestion * 10) - float_suggestion * 10) < 1e-3
```

Comment:
This test will be silently omitted if we change/add step parametrization.

Reply:
Thank you for heads up! Given the step parametrization remains unchanged, I think this part is fine but kindly let me know if I missed anything.

Reply:
I think we can just simply do
```suggestion
    if float_dist.step is not None:
        assert abs(int(float_suggestion * 10) - float_suggestion * 10) < 1e-3
```
Also, `int` always rounds down to the nearest integer, so i guess this test will always pass since `0.0 <= float_suggestion - int(float_suggestion) < 1.0` is always `True`

Reply:
If I understood correctly, this test ensures that `float_suggestion` is at most one decimal place when the `FloatDistribution` given `step=0.1` (In other word, test fails let's say when `float_suggestion = 1.23`).
Hence, I think the condition branch should be explicitly `== 0.1`.

Please kindly let me know if I misunderstood any point.

Reply:
Ok, let's leave it for now then and maybe revisit if parametrization changes. :+1: